source/documentation/values/calculations.html.md.erb

```diff
@@ -20,6 +20,16 @@ introduction: >
   [plain CSS function]: ../at-rules/function#plain-css-functions
 <% end %>
 
+<% impl_status feature: 'Other functions', dart: '1.40.0', libsass: false, ruby: false do %>
+  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.64.0 *always* parse
+  the following [math functions] as Sass functions. To create a plain CSS calculation function
+  call for those implementations, you can write something like
+  `sqrt(#{$number})` instead.
+
+  [special functions]: ../syntax/special-functions
+  [math functions]: https://www.w3.org/TR/css-values-4/#math
+<% end %>
```

Comment:
This seems to be redundant with the `impl_status`es for the `abs()` and `round()` sections below.